Build/QIR_validation.ps1

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Minor, but our other build scripts have generally had all-lowercase names separated by dashes with the relevant verb first (e.g.: `validate-qir.ps1` or `test-qir.ps1`).

Reply:
done

Reply:
changed to `validate-qir.ps1`